docker/rust-all.Dockerfile

```diff
@@ -119,9 +117,9 @@ FROM debian-base AS faucet
 RUN apt-get update && apt-get install -y libssl1.1 ca-certificates nano net-tools tcpdump iproute2 netcat \
     && apt-get clean && rm -r /var/lib/apt/lists/*
 
-RUN mkdir -p /opt/aptos/bin  /aptos/client/data/wallet/
+RUN mkdir  /aptos/client/data/wallet/
```

Comment:
where's your -p?

I think we use /opt/aptos for configs, so if we don't make it, bad things /might/ happen.

Reply:
I don't think that's a problem since docker's `-v` flag which we use to mount those config files automatically creates the target directory if it doesn't yet exist in the docker image.